packs/bit-bucket/actions/archive_repo.yaml

```diff
@@ -0,0 +1,10 @@
+name: archive_repo
+runner_type: run-python
+description: Download a repository as an archive
```

Comment:
We don't have a schema for the result object right now so it would be good to indicate that this action returns a path to the downloaded archive on success.